### ISSUE TITLE
ocf2ocx crashes

### ISSUE THREAD

**Issue by pjohnmeyer** [captable/ocx#46]
### Steps to reproduce

Run

    ocf2ocx ./test/fixtures/example-inc

or

    npm run cli ./test/fixtures/example-inc

### Expected behavior

Normal Excel output

### Actual behavior

```
pj@LTSE-Q293FVD2T0 ocx % ocf2ocx ./test/fixtures/example-inc
node:internal/modules/cjs/loader:998
  throw err;
  ^

Error: Cannot find module 'src/logging'
Require stack:
- /Users/pj/code/ocx/dist/src/model/calculations.js
- /Users/pj/code/ocx/dist/src/model/index.js
- /Users/pj/code/ocx/dist/src/index.js
- /Users/pj/code/ocx/dist/src/cli/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
    at Function.Module._load (node:internal/modules/cjs/loader:841:27)
    at Module.require (node:internal/modules/cjs/loader:1067:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/Users/pj/code/ocx/dist/src/model/calculations.js:9:35)
    at Module._compile (node:internal/modules/cjs/loader:1165:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1219:10)
    at Module.load (node:internal/modules/cjs/loader:1043:32)
    at Function.Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1067:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/Users/pj/code/ocx/dist/src/model/index.js:7:24)
    at Module._compile (node:internal/modules/cjs/loader:1165:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1219:10)
    at Module.load (node:internal/modules/cjs/loader:1043:32)
    at Function.Module._load (node:internal/modules/cjs/loader:878:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/pj/code/ocx/dist/src/model/calculations.js',
    '/Users/pj/code/ocx/dist/src/model/index.js',
    '/Users/pj/code/ocx/dist/src/index.js',
    '/Users/pj/code/ocx/dist/src/cli/index.js'
  ]
}
````

### What OS and version are you running on?

MacOS 12.6

### What Node version(s) are you using?

16.19

**Comment by daviidy** [comment]
That's weird, because it works fine on my side

**Comment by daviidy** [comment]
I fixed it, but got another error, after running the command

`[ERROR] worksheet.nextRow is not a function`
Looking into this currently

**Comment by daviidy** [comment]
Fixed in PR #45
